content/gocourse-2019-05-melbourne.slide

```diff
@@ -894,6 +893,34 @@ Embedding types introduces the possiblity for naming conflicts. Go follows some
 - Deferred functions are called [[https://goplay.space/#ZX5S9xyOS57][in reverse order]]
 - Most commonly used for [[https://goplay.space/#mh9MhO9rDXt][resource cleanup]] (close IO resources, release locks etc.)
 
+* Panic and Recover
+
+- `panic` and `recover` are builtin functions
+- `panic` and `recover` behave similar to `try/catch` in other languages
```

Comment:
sorry, I only just noticed this - try/catch should probably be throw and catch, rather than try/catch. (the `try` would be `defer` I guess), but the meaning is still there, so up to you.